azkaban-common/src/main/java/azkaban/executor/container/watch/KubernetesWatch.java

```diff
@@ -114,7 +139,8 @@ protected void initializePodWatch() throws ApiException {
               null,
               null,
               null,
-              true,
+              null,
```

Comment:
what's the change in this method signature? please double check if the parameter values are aligning with the method in the older client version.

Reply:
There's a new parameter called resourseVersionMatch which would also be null since the resourceVersion is null. Since we're passing one more parameter here, the true for the watch boolean is moved ahead.